examples/ingress-recommendations/README.md

```diff
@@ -0,0 +1,175 @@
+# Ingress recommendations
+
+This example scenario showcases the BotKube recommendations feature around Ingress resources.
+
+## Prerequisites
+
+### Software
+
+Install the following applications:
+
+- [colima](https://github.com/abiosoft/colima)
+- [Helm](https://helm.sh/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Mkcert](https://github.com/FiloSottile/mkcert)
+
+### Set up local Kubernetes cluster
+
+Run the command:
+
+```bash
+colima start --runtime containerd --kubernetes --cpu 2 --memory 3 --disk 20 --profile botkube
+```
+
+### Install Ingress Nginx Controller
+
+Run the command:
+
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace --wait \
+  -f ./examples/ingress-recommendations/ingress-overrides.yaml
+```
+
+### Generate and use self-signed certificate
+
+1. Generate the certificate:
+
+  ```bash
+  mkcert "*.botkube.local" botkube.local localhost 127.0.0.1 ::1
+  ```
+
+1. Create Kubernetes secret:
+
+  ```bash
+  CERT_CRT=$(echo -n "$(cat ./_wildcard.botkube.local+4.pem)" | base64)
+  CERT_KEY=$(echo -n "$(cat ./_wildcard.botkube.local+4-key.pem)" | base64)
+  cat > /tmp/secret.yaml << ENDOFFILE
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: default-ssl-cert
+    namespace: ingress-nginx
+  type: kubernetes.io/tls
+  data:
+    tls.crt: ${CERT_CRT}
+    tls.key: ${CERT_KEY}
+  ENDOFFILE
+
+  kubectl apply -n ingress-nginx -f /tmp/secret.yaml
+  ```
+
+1. Add entry to `/etc/hosts`:
+
+  ```bash
+  readonly DOMAIN="botkube.local"
+  readonly BOTKUBE_HOSTS=("example")
+
+  LINE_TO_APPEND="127.0.0.1 $(printf "%s.${DOMAIN} " "${BOTKUBE_HOSTS[@]}")"
+  HOSTS_FILE="/etc/hosts"
+
+  grep -qF -- "$LINE_TO_APPEND" "${HOSTS_FILE}" || (echo "$LINE_TO_APPEND" | sudo tee -a "${HOSTS_FILE}" > /dev/null)
+  ```
+
+1. Navigate to https://example.botkube.local/ - you should see 404 error, but the connection should be secured.
+
+### Deploy BotKube
+
+1. Export required environment variables:
+
+  ```bash
+  export SLACK_BOT_TOKEN="{token}"
+  export SLACK_CHANNEL="{channel}" # e.g. general
+  ```
+
+1. Deploy BotKube:
+
+  ```bash
+  helm install botkube --namespace botkube ./helm/botkube -f ./examples/ingress-recommendations/botkube-values.yaml --set communications.default-group.slack.token=$SLACK_BOT_TOKEN --set communications.default-group.slack.channels.default.name=$SLACK_CHANNEL --wait --create-namespace 
+  ```
+
+### Deploy example app
+
+Deploy the app with Terminal:
+
+```
+kubectl apply -f ./examples/ingress-recommendations/deploy
+```
+
+## Scenario
+
+In this scenario, we will expose the example application under the `https://example.botkube.local/meme` endpoint.
+
+  See if the URL works: [https://example.botkube.local/meme](https://example.botkube.local/meme). You should still see 404 error.
+
+1. To expose the app, we need an Ingress resource. Create it with BotKube:
```

Comment:
```
... we need an Ingress resource which you can create with BotKube as follows.
```

Reply:
Changed to "To expose the app, we need an Ingress resource. Post the following command on Slack to create it with BotKube:", as it would be a quite long sentence. Hope you'll like it 👍